tgui/packages/tgui/interfaces/EmotePanel.tsx

```diff
@@ -23,47 +24,23 @@ export const EmotePanelContent = (props) => {
   const { act, data } = useBackend<EmotePanelData>();
   const { emotes } = data;
 
-  const [filterVisible, toggleVisualFilter] = useLocalState<boolean>(
-    'filterVisible',
-    false,
-  );
+  const [filterVisible, toggleVisualFilter] = useState<boolean>(false);
```

Comment:
several in this file

Reply:
Applied via 0fbd285de3fc5e9ab8da6d6b196dece3150075da